src/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\HttpCacheBundle\Exception;
+
+use Neusta\Pimcore\HttpCacheBundle\Cache\CacheType;
+
+final class InvalidArgumentException extends \InvalidArgumentException implements PimcoreHttpCacheException
+{
+    public static function becauseEmptyCacheTagIsNotAllowed(): self
```

Comment:
There is `becauseCacheTypeIsEmpty`. What about changing this one to `becauseCacheTagIsEmpty`?